Tests/API/ApiTests.cs

```diff
@@ -596,6 +596,47 @@ public void BacktestingData_CanBeDownloadedAndSaved_Successfully()
             Assert.IsTrue(File.Exists(minutePath));
         }
 
+        /// <summary>
+        /// Test read price API for given symbol
+        /// </summary>
+        [Test]
+        public void ReadPriceWorksCorrectlyFor1Symbol()
+        {
+            var spy = Symbol.Create("SPY", SecurityType.Equity, Market.USA);
+            var pricesList = _api.ReadPrices(new [] { spy });
+
+            Assert.IsTrue(pricesList.Success);
+            Assert.AreEqual(pricesList.Prices.Count, 1);
+
+            var price = pricesList.Prices.First();
+            Assert.AreEqual(price.Symbol, spy.ID.ToString());
+            Assert.AreNotEqual(price.Price, 0);
+            var updated = price.Updated;
+            var reference = DateTime.UtcNow.Subtract(TimeSpan.FromDays(3));
+            Assert.IsTrue(updated > reference);
+        }
+
+        /// <summary>
+        /// Test read price API for multiple symbols
+        /// </summary>
+        [Test]
+        public void ReadPriceWorksCorrectlyForMultipleSymbols()
+        {
+            var spy = Symbol.Create("SPY", SecurityType.Equity, Market.USA);
+            var aapl = Symbol.Create("AAPL", SecurityType.Equity, Market.USA);
+            var pricesList = _api.ReadPrices(new[] { spy, aapl });
+
+            Assert.IsTrue(pricesList.Success);
+            Assert.AreEqual(pricesList.Prices.Count, 2);
+
+            Assert.IsFalse(pricesList.Prices.Any(x => x.Price == 0));
+            Assert.AreEqual(pricesList.Prices.Count(x => x.Symbol == aapl.ID.ToString()), 1);
+            Assert.AreEqual(pricesList.Prices.Count(x => x.Symbol == spy.ID.ToString()), 1);
+
+            var reference = DateTime.UtcNow.Subtract(TimeSpan.FromDays(3));
+            Assert.IsFalse(pricesList.Prices.Any(x => x.Updated < reference));
```

Comment:
double negative? perhaps `Assert.IsTrue(priceList.Prices.All(x => x.Updated > reference)` is more intutive?

Reply:
Yeah! I'll change it 👍